ui/raidboss/data/06-ew/dungeon/lapis_manalis.txt

```diff
@@ -0,0 +1,283 @@
+1276.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7A9D:/
+1277.7 "Scarecrow Chase 1" sync / 1[56]:[^:]*:Galatea Magna:7FBF:/
+1278.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1279.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7A9D:/
+1280.7 "Scarecrow Chase 2" sync / 1[56]:[^:]*:Galatea Magna:7FBF:/
+1281.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1282.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7A9D:/
+1283.7 "Scarecrow Chase 3" sync / 1[56]:[^:]*:Galatea Magna:7FBF:/
+1284.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1285.9 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7A9D:/
+1286.7 "Scarecrow Chase 4" sync / 1[56]:[^:]*:Galatea Magna:7FBF:/
+1288.6 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1293.6 "Tenebrism" sync / 1[56]:[^:]*:Galatea Magna:7A96:/
+
+#~~~~~~~~~~#
+# Cagnazzo #
+#~~~~~~~~~~#
+
+# -p 79A3:2015.6 799C:2102.9 7986:2176.3
+# -ii 7FB7 7FB5
+# -it "Cagnazzo"
+
+2000.0 "--sync--" sync / 00:0839::Deepspine will be sealed off/ window 2000,1
+2015.6 "Stygian Deluge" sync / 1[56]:[^:]*:Cagnazzo:79A3:/
+2025.4 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:798F:/
+2038.0 "Antediluvian" sync / 1[56]:[^:]*:Cagnazzo:7990:/
+2042.0 "Antediluvian" sync / 1[56]:[^:]*:Cagnazzo:7990:/
+2043.3 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7991:/
+2044.3 "Body Slam" sync / 1[56]:[^:]*:Cagnazzo:(7992|7993):/
+2046.0 "Antediluvian" sync / 1[56]:[^:]*:Cagnazzo:7990:/
+2051.1 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:799B:/
+2060.9 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7FB4:/
+2062.8 "Hydraulic Ram" #sync / 1[56]:[^:]*:Cagnazzo:7FB6:/
+2063.2 "Hydrobomb" #sync / 1[56]:[^:]*:Cagnazzo:7FB8:/
+2063.9 "Hydraulic Ram" #sync / 1[56]:[^:]*:Cagnazzo:7FB6:/
+2064.3 "Hydrobomb" #sync / 1[56]:[^:]*:Cagnazzo:7FB8:/
+2065.0 "Hydraulic Ram" #sync / 1[56]:[^:]*:Cagnazzo:7FB6:/
+2065.4 "Hydrobomb" #sync / 1[56]:[^:]*:Cagnazzo:7FB8:/
+2066.1 "Hydraulic Ram" #sync / 1[56]:[^:]*:Cagnazzo:7FB6:/
+2066.5 "Hydrobomb" #sync / 1[56]:[^:]*:Cagnazzo:7FB8:/
+2067.2 "Hydraulic Ram" #sync / 1[56]:[^:]*:Cagnazzo:7FB6:/
+2067.6 "Hydrobomb" #sync / 1[56]:[^:]*:Cagnazzo:7FB8:/
+2068.3 "Hydraulic Ram" #sync / 1[56]:[^:]*:Cagnazzo:7FB6:/
+2068.7 "Hydrobomb x2" #sync / 1[56]:[^:]*:Cagnazzo:7FB8:/
+2069.4 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7996:/
+2073.5 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:799B:/
+2084.6 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7A8F:/
+2085.2 "Hydrofall" sync / 1[56]:[^:]*:Cagnazzo:7A90:/
+2095.9 "Cursed Tide" sync / 1[56]:[^:]*:Cagnazzo:799A:/
+
+2099.5 "--untargetable--"
+2099.5 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:799B:/
+2102.9 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:799C:/
+2113.7 "Neap Tide" sync / 1[56]:[^:]*:Cagnazzo:799E:/
+2118.1 "Hydrovent" sync / 1[56]:[^:]*:Cagnazzo:79A0:/
+2124.7 "Spring Tide" sync / 1[56]:[^:]*:Cagnazzo:799F:/
+2134.1 "Hydrovent" sync / 1[56]:[^:]*:Cagnazzo:79A0:/
+2145.1 "Hydrovent" sync / 1[56]:[^:]*:Cagnazzo:79A0:/
+2146.4 "--sync--" sync / 00:0044:Cagnazzo:No more games!/ window 30,30
```

Comment:
In general I'd prefer not to use chat lines for syncs.  Is it possible to just add `14:` line sync for Tsunami with a larger window?

Can you also add in some extra time here? This is an add phase with variable time.  Usually I just bump up to some round number, e.g. 2500.  See: https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/06-ew/alliance/euphrosyne.txt#L345-L358

Reply:
If we sync on Tsunami there's no way to have a warning callout for the raidwide since it's not casted, it's an instant.

Reply:
Is there anything else to sync on here?

Reply:
He spams `Hydrovent` on the environment until the `Fearsome Flotsam` spears die, and then goes into the transition with the chatline.  There's really nothing outside of the Tsunami itself.

Reply:
You need to adjust these times too, I think?

Have you run this through `util/log_tools/test_timeline.ts`? It's the same `-f` and `-lf` params, but you also need to pass `-t lapis_manlis`.

Reply:
The first pass was:
```
2146.4 "--sync--" sync / 00:0044:Cagnazzo:No more games!/ window 30,30
2159.1 "Tsunami" sync / 1[56]:[^:]*:Cagnazzo:79A1:/ window 60,60

2163.1 "--targetable--"
2169.2 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:799B:/
2176.3 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7986:/
```

This is now:
```
2146.4 "--sync--" sync / 00:0044:Cagnazzo:No more games!/ window 30,30
2159.1 "Tsunami" sync / 1[56]:[^:]*:Cagnazzo:79A1:/ window 60,60

2500.0 "--targetable--"
2506.1 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:799B:/ window 500,0
2513.2 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7986:/
```

I think it should be this, right? Tsunami starts the next phase?
```
2483.3 "--sync--" sync / 00:0044:Cagnazzo:No more games!/ window 400,0
2496.0 "Tsunami" sync / 1[56]:[^:]*:Cagnazzo:79A1:/ window 400,0
2500.0 "--targetable--"
2506.1 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:799B:/
2513.2 "--sync--" sync / 1[56]:[^:]*:Cagnazzo:7986:/
```

Reply:
I forgot the sync when I bumped the timeline up before so I just threw it on the first sync after he was targetable.  I can just move it to Tsunami.